content/en/docs/setup/best-practices/certificates.md

```diff
@@ -127,6 +132,15 @@ Same considerations apply for the service account key pair:
 |  sa.key                      |                             | kube-controller-manager | --service-account-private-key-file   |
 |                              | sa.pub                      | kube-apiserver          | --service-account-key-file           |
 
+The following diagram illustrates the files from the previous tables you need to provide if you are generating all of your own keys and certificates:
+
+
+![Generate all certificates yourself](/images/docs/Certs-AllCerts.svg)
+
+Flags in the following diagram illustrate which Kubernetes control plane services and commands consume each certificate and key. Refer to the previous table to see which options are needed to pass each file.
+
+![Certificates and keys passed to commands and services](/images/docs/Certs-AllFlags.svg)
```

Comment:
In order to make this graph a little bit simpler, I would suggest to not include the representation of the tree structure but instead to focus only on components and certificates.
Unfortunately, the relation components<->certificates is a matrix,  so it is not easy to find  a simple representation (might be venn diagrams?), so I would be ok if this details exists only in the table above

Reply:
Okay. I'll take out this diagram and consider working on something like a venn diagram after this PR is merged.